src/SelectableBox/SelectableBoxSet.jsx

```diff
@@ -19,6 +19,7 @@ const SelectableBoxSet = React.forwardRef(({
   type,
   columns,
   className,
+  ...props
```

Comment:
[non-blocking suggestion] I wonder if it's worth adding a test case to assert that prop spreading is supported (e.g., in case the line were to get removed erroneously somehow)?

Reply:
Good callout! Quick question on this, though. I notice the tests for this component are using `enzyme`. Is Paragon also planning to migrate to React 18 (I would think, yes)? And if so, will we also be following this [proposed OEP](https://github.com/openedx/open-edx-proposals/blob/de17de9aee40a2baa29cd1a87c1640fca2586489/oeps/best-practices/oep-0011/decisions/0001-enzyme-deprecation.rst)?

Reply:
We've decided we will not be using `react-unit-test-utils` in the Paragon repo for shallow/snapshot testing. As you noted, Raccoon Gang is actively migrating away from Enzyme tests in favor of RTL / `react-test-renderer` (where appropriate). Generally, my two cents is that explicit assertions are generally preferable than snapshots.

I would recommend using RTL to render the component with an `data-testid` prop and then assert that `expect(screen.getByTestId(...)).toBeInTheDocument()`.

Reply:
I was not able to get the tests working with `toBeInTheDocument()`. I was seeing this error:
`TypeError: expect(...).toBeInTheDocument is not a function`

However, `toBeTruthy()` was able to run the assertion just fine. I'm happy to change this but may need some guidance if we have to use `toBeInTheDcoument`.

Reply:
Ah, @MaxFrank13: I believe that error is likely due to not including an import for the following in the test file:

```js
import '@testing-library/jest-dom/extend-expect';
```